app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -55,24 +65,60 @@ class PipelineSampleReport extends React.Component {
       }
       */
       excluded_categories: (cached_cats) ? JSON.parse(cached_cats) : [],
-      name_type: cached_name_type ? cached_name_type : 'scientific',
+      name_type: cached_name_type ? cached_name_type : 'Scientific Name',
       search_taxon_id: 0,
       rendering: false,
-      loading: true
-    };
-
-    this.applyNameType = this.applyNameType.bind(this);
+      loading: true,
+      activeThresholds: this.defaultThresholdValues,
+      allThresholds: [
+        {
+          name: 'Score',
+          value: 'NT_aggregatescore'
+        }, {
+          name: 'NT Z Score',
+          value: 'NT_zscore'
+        }, {
+          name: 'NT rPM',
+          value: 'NT_rpm'
+        }, {
+          name: 'NT r (total reads)',
+          value: 'NT_r'
+        }, {
+          name: 'NT %id',
+          value: 'NT_percentidentity'
+        }, {
+          name: 'NT log(1/e)',
+          value: 'NT_neglogevalue'
+        }, {
+          name: 'NT %conc',
+          value: 'NT_percentconcordant'
+        }, {
+          name: 'NR Z Score',
+          value: 'NR_zscore'
+        }, {
+          name: 'NR r (total reads)',
+          value: 'NR_r'
+        }, {
+          name: 'NR rPM',
+          value: 'NR_rpm'
+        }, {
+          name: 'NR %id',
+          value: 'NR_percentidentity'
+        }, {
+          name: 'R log(1/e)',
+          value: 'NR_neglogevalue'
+        }, {
+          name: 'NR %conc',
+          value: 'NR_percentconcordant'
+        }]
+      };
     this.applySearchFilter = this.applySearchFilter.bind(this);
-    this.applyThresholdFilters = this.applyThresholdFilters.bind(this);
```

Comment:
why do you remove this? This still exists.

Reply:
We were binding the `this` context to the method, but we have too many binds already and  we could avoid that by using the arrow functions like this 
`<button onClick={ () => this.methodName() }></button>` then  we don't have to call the bind method again